flask_node:
  ros__parameters:
    # The path to the directory containing offline map tile sets.
    # This path is relative to the package's share directory.
    # Example structure: <install_space>/share/mvp_gui_2/mvp_gui_offline_map/
    tiles_dir: "mvp_gui_offline_map/"
    db_filename: "mvp_gui.db" # Database will be stored in the package share dir
    kml_upload_folder: "kml_uploads" # Also in package share dir

ros_interface_node:
  ros__parameters:
    #----------------------------
    # Topics
    #----------------------------
    topic_ns: "/mvp2_test_robot/remote/id_2/"
    ## pose
    poses_source: "odometry"
    geo_pose_source: "geopose"
    topside_geo_pose_source: "topside/geopose"
    secondary_geo_pose_source: "secondary/geopose"
    survey_geopath_source: "survey/geopath"
    ## state
    helm_state_get: "helm/state"
    controller_state_get: "controller_state"
    gpio_power_get: "gpio_power_state"
    roslaunch_state_get: "roslaunch_state"
    ## Info
    computer_info_source: "computer_info"
    power_info_source: "power_info"

    #----------------------------
    # Services
    #----------------------------
    service_ns: "/mvp2_test_robot/remote/id_2/"
    helm_state_set: "mvp_helm/change_state"
    controller_state_set: "controller/set"
    pub_waypoints_service: "mvp_helm/set_waypoints"
    set_power_service: "gpio_manager/set_power/"
    
    # Name of the node to fetch dynamic launch/power configurations from
    c2_commander_node: "/mvp2_test_robot/mvp_c2_commander"